core-bundle/src/Controller/SitemapController.php

```diff
@@ -76,6 +78,7 @@ public function __invoke(Request $request): Response
         $urlSet = $sitemap->createElementNS('https://www.sitemaps.org/schemas/sitemap/0.9', 'urlset');
 
         foreach ($urls as $url) {
+            $url = htmlspecialchars($url, ENT_XML1, $this->encoding, false);
             $loc = $sitemap->createElement('loc', $url);
```

Comment:
```suggestion
            $loc = $sitemap->createElement('loc');
            $loc->appendChild($sitemap->createTextNode($url));
```
I think we should use the correct DOM methods here instead of `htmlspecialchars`.